pycbc/mchirp_area.py

```diff
@@ -12,6 +12,51 @@
 from scipy.integrate import quad
 
 
+def insert_args(parser):
+    mchirp_group = parser.add_argument_group("Arguments for computing the "
+                                             "areas of the CBC regions using "
+                                             "mchirp and effective distance.")
+    mchirp_group.add_argument('--max-m1', type=float, default=45.0,
```

Comment:
The option name and help string could be a bit more descriptive (comment applies to all the options). Imagine running `pycbc_live --help` and seeing an option called `--max-m1` with help "Maximum value for m1"... What is m1? What is this maximum value used for? Maybe `--src-class-m1-range 1 45` is less obscure.

Also, this makes me think that we may want to switch to `.ini` files instead of hundreds of possible options at some point.

Reply:
that's not an issue that this change has caused and not one we should be addressing as part of this PR .. i.e. 'you only just started thinking this?'

Reply:
Yeah nevermind, just thinking loud. The first paragraph is the point to focus on.

Reply:
The first paragraph of the initial comment still stands...